src/app/wallet/proposals/proposal-details/proposal-details.component.ts

```diff
@@ -43,25 +43,43 @@ export class ProposalDetailsComponent implements OnInit, OnDestroy {
       showLabels: false,
       donut: true,
       legend: {
-        maxKeyLength: 35,
+        maxKeyLength: 100,
         margin: {
           left: 0,
           right: 0,
           top: 5,
           bottom: 5
         },
-        // padding: {
-        //   top: 0,
-        //   bottom: 0,
-        //   right: 5,
-        //   left: 5
-        // }
+        padding: {
+          top: 0,
+          bottom: 0,
+          right: 5,
+          left: 5
+        }
       },
       color: ['#02E8B0', '#ec4b50', '#108cda', '#f1cc00', '#7e6c95'], // green, red, blue, yellow, violet
       tooltip: {
         enabled: true,
         hideDelay: 0,
         useInteractiveGuideline: false
+      },
+      callback: function() {
```

Comment:
This function doesn't seem to work 100% for wrapping the text. Particularly when resizing the application window (I'm noticing a significant amount of text overlap).

If this is not particularly possible or easy to implement, then lets leave it for now and address it as part of a different issue.

Reply:
@zaSmilingIdiot Is that is the known issue and even if you click on dots the text field again rendered due to which it showing the breaking text in one line again which is weird for me 

Thanks

Reply:
Okay, so because its a known issue, then we either need to correct it or remove it. 

I don't see a need to waste unnecessary amounts of time to try to resolve this right now. If its not resolvable in a few hours, then perhaps log the issue as a separate one (with a question of replacing the graphing library), remove this update function, and let the tooltip for each legend option suffice for the immediate future. Particularly since the expansion of the long text was not part of this original request.